client/jetpack-cloud/sections/partner-portal/select-partner-key/index.tsx

```diff
@@ -44,29 +49,37 @@ export default function SelectPartnerKey(): ReactElement | null {
 	}, [ hasKey ] );
 
 	return (
-		<>
+		<Main className="select-partner-key">
 			<QueryJetpackPartnerPortalPartner />
-			<Main className="select-partner-key">
-				{ isFetching && (
-					<Card>
-						<Spinner />
-					</Card>
-				) }
 
-				{ ! isFetching && keys.length === 0 && (
-					<Card>{ translate( 'You are not registered as a partner.' ) }</Card>
-				) }
+			<CardHeading size={ 36 }>{ translate( 'Partner Portal' ) }</CardHeading>
 
-				{ ! isFetching &&
-					keys.map( ( key ) => (
-						<Card key={ key.id } className="select-partner-key__card">
+			{ isFetching && <Spinner /> }
+
+			{ showError && (
+				<div className="select-partner-key__error">
+					<p>{ translate( 'Your account is not registered as a partner account.' ) }</p>
```

Comment:
I'm wondering if we should mention how they can become a partner. If someone gets to this page without a key, they are likely to know about the Partner Portal.